apps/namadillo/public/config.toml

```diff
@@ -3,3 +3,6 @@
 #rpc_url = ""
 #masp_indexer_url = ""
 #localnet_enabled = false
+
+github_chain_registry_base_url = "https://raw.githubusercontent.com/anoma/namada-chain-registry/refs/heads/main"
+github_namada_interface_url = "https://raw.githubusercontent.com/anoma/namada-interface/refs/heads/main"
```

Comment:
I didn't understand the use case of this 🤔

Reply:
Just in case someone would want to point to their own chain registry for some reason. Probably won't be used